### PR TITLE
Specifications for Event API

### PR DESCRIPTION
**Pull request by mod**
Request for Comments

### CODE REVIEW

File: docs/specs/event_api.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
+# `DRAFT` RabbitMQ Peatio Event API
+
+## Overview of RabbitMQ details
+
+Peatio submits all events into single exchange called `peatio.events`.
```
Comment (mod) [inline docs/specs/event_api.md]: into 3 type of exchange:
peatio.events.system: Any basic events occurred in state
peatio.events.model: Events related to ActiveRecord change of attribute after_commit
peatio.events.market: Events occurred for peatio matcher and notification websocket

File: docs/specs/event_api.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+The routing key (same as event name) consists of two parts: 
+
+  1) category of events or subject, like `deposit`, `withdraw`, `system`.
+
+  2) event name, like `created`, `updated`, `destroyed`, `low_hot_wallet_balance`.
```
Comment (mod) [inline docs/specs/event_api.md]: I would name the routing key
`peatio.system.low_hot_wallet_balance`

or

`barong.model.account.updated`

**Review by mod — COMMENTED** [review]
@yivo does jwt-multisig forces the data: {} field ?
I would skip it in favor of event: {}

Also I would like an example of system event and market event

system can be hot_wallet_low_balance
market can be something trade matcher need to match or trade UI need to pop for updating charts/orderbook

File: docs/specs/event_api.md (hunk 1 above)
Comment (joh90) [inline docs/specs/event_api.md]: should we consider using fanout exchange? These events could be consumed by various consumers

based on your sample code below
```
channel     = session.channel
exchange    = channel.direct(<exchange_name>)
```

**Pull request merged**